install_config/install/prerequisites.adoc

```diff
@@ -547,6 +547,10 @@ post back status, to receive tasks, and so on.
 | *10250*
 |TCP
 |The master proxies to node hosts via the Kubelet for `oc` commands.
+
+| *10010*
+|TCP
+|If using CRI-O, open this port to perform `oc exec` and `oc rsh` operations.
```

Comment:
@ahardin-rh Am I being to picky to suggest s/perform/allow? This almost sound like opening the port does the operations. Or not.....